EIPS/eip-7923.md

```diff
@@ -72,12 +72,15 @@ Benchmarks were performed on a 2019-era CPU, with the ability to `keccak256` aro
 - Time to update a hashmap with 512 items: 8ns
 - Time to update a hashmap with 8192 items: 9ns
 - Time to update a hashmap with 5mm items: 108ns
+- Time to execute the `mmap` syscall: 230ns
 
 These suggest the following prices:
 
 - 100 gas to allocate a page, and
 - 6 gas for a page thrash
 
+Note that the cost to execute `mmap` (~11 gas) is already well-paid for by the base cost of the CALL series of instructions (100 gas).
```

Comment:
This `mmap` is something new in this EIP, so it was not included in the CALL gas before. However, most likely (with my limited knowledge of hardware and memory stuff in physical systems) for a CALL there is also some new state setup ("fresh" EVM call frame) and this likely also allocates some memory. 

My main point here is that it would be more logical to say that the CALL gas should be made higher by 11 gas to represent the new `mmap` costs, but that it is being believed that the 100 gas cost is also sufficient (so this 100 gas is enough, even in the worse case, to cover the base "time" CALL takes to setup the basic stuff)

Reply:
Or, maybe I'm reading this wrong and this is actually what you mean by "already well-paid for" and I understood this wrong.

Reply:
yes, that's what i meant by "already well-paid for"